src/components/Boards/components/BoardTile/BoardTile.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import CardActions from '@material-ui/core/CardActions';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import CardHeader from '@material-ui/core/CardHeader';
+import Avatar from '@material-ui/core/Avatar';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import Card from '@material-ui/core/Card';
+
+interface Props {
+  id: string;
+  name: string;
+}
+
+export default function DefaultListItem(props: Props) {
+  return (
+    <Grid item xs={6} sm={3}>
+      <Card>
+        <CardActionArea>
+          <CardHeader
+            avatar={
+              <Avatar aria-label="recipe">
+                {props.name ? props.name[0].toUpperCase() : ''}
```

Comment:
Question: Do we allow `name` to start with a space?

Reply:
We're not sanitizing anything currently. We'll have to trim on the way in

Reply:
Gotcha, we can ticket this and handle it in the future.